pages/auth.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div>
+    <NuxtPage class="auth-container card" :route="route" />
```

Comment:
since there's style conflicts with omorphia, use `universal-card` for now. something we'll have to deal with but probably out of scope for now. this will fix the padding on the card